src/app/task/task.component.html

```diff
@@ -1,25 +1,28 @@
 <h2>Task To Do: </h2>
 
 <div>
-
     <form [formGroup]="taskForm" (ngSubmit)="addTask()">
-        <label> Name:
-      <input type="text" formControlName="name">
-    </label>
+        <label for="form1" class=''> Name:
+            <input mdbInput type="text" id="form1" class="form-control" formControlName="name">
+        </label>
         <label> Description:
-      <input type="text" formControlName="description">
-    </label>
+            <input mdbInput type="text" id="form1" class="form-control" formControlName="description">
```

Comment:
id of the element should indicate what it is used for. For example for this one it should be description or task-description